client/Lykke.Service.Assets.Client/Models/v3/Asset.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Lykke.Service.Assets.Client.Models.v3
+{
+    public class Asset
+    {
+        public string Id { get; set; }
+        public string DisplayId { get; set; }
+        public int Accuracy { get; set; }
+        public bool IsDisabled { get; set; }
```

Comment:
According to what considerations exactly these fields were chosen for this projection?

Reply:
This is what is strongly related to Asset BC, and needed by the first consumer of this contracts (HFT API). This list of fields will be evolving for some time (as far as new consumers will be added).